sinks/signalfx/signalfx.go

```diff
@@ -17,9 +18,81 @@ import (
 	"github.com/stripe/veneur/trace"
 )
 
+type destinations struct {
```

Comment:
I could use some docs around this to explain the reasons behind these being plucked out. (Problem being that the definition of `collection` and the similarity in words meant that when I got to the usage below I had forgotten wtf it was!

Reply:
Yup, this is extremely confusing. I'll try to untangle.